.github/ISSUE_TEMPLATE/3documentation.yml

```diff
@@ -0,0 +1,31 @@
+name: Documentation issue
+description: Report a possible issue in the documentation
+labels: doc
+body:
+  - type: markdown
+    attributes:
+      value: |
+        Thank you for submitting this issue! We'll try to triage it ASAP!
+        Please do not submit bug reports or feature requests about the package itself.  
+  - type: input
+    id: documentation-url
+    attributes:
+      label: Documentation URL
+      description: Please provide a link to the documentation page where the issue is found
+      placeholder: https://expressjs.com/en/starter/hello-world
```

Comment:
This won't be relevant for the whole organization.

Reply:
removed